### PR TITLE
feat: embed webflow and map

### PR DESCRIPTION
**Pull request by KobbyMmo**
## Description

Embed Webflow Sensors and Check if client Id is set in browser storage.
When reviewing, please check the development flow for web flow and the map.

Fixes # (issue)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Screenshots
image

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👏🏾 @KobbyMmo 

---------------------------

I think `src/pages/dashboard` still has precedence over `public/dashboard` (I had to add `/index.html` explicitly to make sure the one in public is called). Did you also experience this  and if so, does Nextjs say anything about it?

**Comment by kilemensi** [comment]
PS: Have we also tried serving the map from the `public` folder as well e.g. from `/public/map`? We may be able to cut down our deployments to just one.

**Comment by KobbyMmo** [comment]
> I think `src/pages/dashboard` still has precedence over `public/dashboard` (I had to add `/index.html` explicitly to make sure the one in public is called). Did you also experience this and if so, does Nextjs say anything about it?

You always have to add the extension to access the files in the public directory. They talk of files having the [same](https://nextjs.org/docs/basic-features/static-file-serving) name but extensions [differentiate](https://nextjs.org/docs/messages/conflicting-public-file-page) them

**Comment by kilemensi** [comment]
> > I think `src/pages/dashboard` still has precedence over `public/dashboard` (I had to add `/index.html` explicitly to make sure the one in public is called). Did you also experience this and if so, does Nextjs say anything about it?
> 
> You always have to add the extension to access the files in the public directory. They talk of files having the [same](https://nextjs.org/docs/basic-features/static-file-serving) name but extensions [differentiate](https://nextjs.org/docs/messages/conflicting-public-file-page) them

Think @KobbyMmo , think... The point of that question was to make you think: _technically_ yes, there is a problem but as far as our users are concerned, there shouldn't be. Based on Nextjs docs and our setup, there should be ways we can go **around** this, right?

**Comment by KobbyMmo** [comment]
> Think @KobbyMmo , think... The point of that question was to make you think: _technically_ yes, there is a problem but as far as our users are concerned, there shouldn't be. Based on Nextjs docs and our setup, there should be ways we can go **around** this, right?

@kilemensi  Yes, there are lots of ways to get around that for sure, that was an answer to your question though.  Off my head, I think we can redirect from a dummy page, serve the page from an API endpoint, remove the extension, and set the `content-type` manually. I am also prety sure nextjs might offer some redirection from their config pages

**Review by kilemensi — COMMENTED** [review]
QQ @KobbyMmo ... Is there any reason why we're not "integrating" map directly under `src` i.e. shouldn't existing as it's own "git" repo with build scripts, etc.?

**Comment by KobbyMmo** [comment]
@kilemensi  The map is served as an Html Page and it's in Vanilla js. The .git files shouldn't be there, for the scripts, I have included build scripts from the parent project.

It uses lots of dom-manipulation and I wanted to keep the code as close to the original map as possible, while still allowing us to manipulate the map in the current workspace.

We will still have to build the map separately and attach the build files to the public folder 
so now, when you run yarn, it installs for both, and when you run yarn dev, it builds the js and stores the build in the public for use by whoever requires it.

It's definitely possible to adapt the map in the src, but I think it's not worth it now.

File: public/js/counties.js [hunk 1]
```diff
@@ -0,0 +1,480 @@
...
+    country: 'kenya',
+    zoom: '9',
+    center: '-0.56333,34.93583',
+  },
+};
+
+window.onload = function() {
+  const countySelect = document.getElementById("county-select");
+  Object.keys(countiesLocation).forEach(key => {
+  const c = new Option(countiesLocation[key].label, key)
```
Comment (gertie-sheshe) [inline public/js/counties.js:482]: Can we give it another name than `c`? 😅

**Review by nerdlet — COMMENTED** [review]
@KobbyMmo Looks good.

**Review by kilemensi — APPROVED** [review]
🚀 @KobbyMmo 

-------------------------

1. Use Donut or Pie chart but don't mix them,
2. Donut/Pie chart are usually smaller and centred rather than taking the whole chart container size (the chart itself, not title or legend). Can we make them may be half or two thirds the of their current size? 
Screenshot from 2021-04-12 16-11-36
vs
Screenshot_2021-04-09 Material Design – Data visualization

3. Too much spacing between the last chartss descriptions and resources components
Screenshot from 2021-04-12 16-12-12
4. On mobile, selecting a different county should close the modal. This doesn't work, I have to manually dismiss the modal (in Firefox anyways).
Peek 2021-04-12 16-18

**Comment by kilemensi** [comment]
PS: @KobbyMmo , are we deleting the stuff we no longer need? For example `/pages/dashboard`.

**Pull request merged**